mkt/commonplace/templates/commonplace/includes/open_graph.html

```diff
@@ -1,8 +1,8 @@
 {% if app %}
-  <meta property="og:title" content="{{ app.name }}">
+  <meta property="og:title" content="{{ app.name|e }}">
   <meta property="og:type" content="website">
   <meta property="og:image" content="{{ app.get_icon_url(64) }}">
-  <meta name="description" content="{{ app.description }}">
+  <meta name="description" content="{{ app.description|striptags|e|truncate(200)|replace('"', '&quot;') }}">
```

Comment:
Okay, I think I was confused by the test after it goes through pyquery. I'll fix.